lib/rspotify/user.rb

```diff
@@ -38,6 +38,16 @@ def self.refresh_token(user_id)
       response = RestClient.post(TOKEN_URI, request_body, RSpotify.send(:auth_header))
       response = JSON.parse(response)
       @@users_credentials[user_id]['token'] = response['access_token']
+      proc = @@users_credentials[user_id]['access_refresh_callback']
```

Comment:
In Ruby, proc is a reserved keyword and an alias for Proc.new
> proc { |...| block } → a_proc
> Equivalent to Proc.new.
> https://ruby-doc.org/core-2.6/Kernel.html#method-i-proc

```suggestion
      access_refresh_proc = @@users_credentials[user_id]['access_refresh_callback']
```